src/schema/v2/object_identification.ts

```diff
@@ -58,15 +58,32 @@ const SupportedTypes: any = {
     "./sale",
     "./sale_artwork",
     "./user",
+    "./homeView/index",
   ],
 }
 
 const typeNames = {
   "./filterArtworksConnection": "filterArtworksConnection",
+  "./homeView/index": "Section",
```

Comment:
We can avoid this (I think) if we move `Section` (from the file above) to a separate file and `export default` it.

Reply:
nice! this helps avoid the conversion to `globalID` and `internalID/id` I guess?

Reply:
It's probably fine to do without the default export and have the mapping here (that's why this exists, so you don't need to do default export).